tavern/graphql/query_test.go

```diff
@@ -0,0 +1,74 @@
+package graphql_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/99designs/gqlgen/client"
+	"github.com/99designs/gqlgen/graphql/handler"
+	"github.com/kcarretto/realm/tavern/ent/enttest"
+	"github.com/kcarretto/realm/tavern/graphql"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestUsersQuery ensures that the users query works as expected.
+func TestUsersQuery(t *testing.T) {
+	// Initialize Test Context
+	ctx := context.Background()
+
+	// Initialize DB Backend
+	graph := enttest.Open(t, "sqlite3", "file:ent?mode=memory&cache=shared&_fk=1")
+	defer graph.Close()
+
+	// Initialize sample data
+	testUser := graph.User.Create().
+		SetName("bobdylan").
+		SetIsActivated(false).
+		SetIsAdmin(true).
+		SetOAuthID("likearollingstone").
+		SetPhotoURL("https://upload.wikimedia.org/wikipedia/commons/0/02/Bob_Dylan_-_Azkena_Rock_Festival_2010_2.jpg").
```

Comment:
Any chance we can use a file we manage maybe in the realm repo?

Reply:
Doesn't do anything with URL can be garbage string.

Reply:
File is not actually used at all, so this change is unnecessary